src/plugins/drawing/drawing-registration.ts

```diff
@@ -46,11 +57,29 @@ registerDrawingObjectInfo({
 });
 
 registerDrawingToolInfo({
-  name: "group",
-  buttonComponent: GroupObjectsButton
+  name: "group"
 });
 
 registerDrawingToolInfo({
-  name: "ungroup",
-  buttonComponent: UngroupObjectsButton
+  name: "ungroup"
 });
+
+registerTileToolbarButtons("drawing", [
+  { name: "select", component: SelectButton },
+  { name: "line", component: LineButton },
+  { name: "vector", component: VectorButton },
+  { name: "rectangle", component: RectangleButton },
+  { name: "ellipse", component: EllipseButton },
+  { name: "stamp", component: StampButton },
+  { name: "stroke-color", component: StrokeColorButton },
+  { name: "fill-color", component: FillColorButton },
+  { name: "text", component: TextButton },
+  { name: "upload", component: ImageUploadButton },
+  { name: "group", component: GroupButton },
+  { name: "ungroup", component: UngroupButton },
+  { name: "duplicate", component: DuplicateButton },
+  { name: "delete", component: DeleteButton },
+  { name: "new-variable", component: NewVariableButton},
+  { name: "insert-variable", component: InsertVariableButton},
+  { name: "edit-variable", component: EditVariableButton},
```

Comment:
These should not be registered here -- we don't want a dependency from drawing --> shared variables, only the other way.

You should be able to simply call `registerTileToolbarButtons` again from `shared-variables-registration.ts`